api/crud.py

```diff
@@ -261,10 +261,6 @@ async def products_add_related(items: Iterable[models.Product]):
     return items
 
 
```

Comment:
this may be legacy code. api.views.get_products is used instead.

Reply:
Oh indeed, you're right. Thank you